Header header
geometry_msgs/Vector3 p_current
geometry_msgs/Vector3 p_nearest
string assistance_mode
float64 d_nearest
float64 k_value
int64 trajectory_id
